### DISCUSSION TITLE
App deposit can no longer be found.

### DISCUSSION THREAD

**Discussion by 5a5ch4** [MustardChef/WSABuilds#354] [Q&A]
Hello, I have installed the Versi[WSA_2311.40000.5.0_x64_Release-Nightly-MindTheGapps-13.0.7" and WSA-Packman to add apk-filein.

But unfortunately, after restarting, I get an error message when I want to open an app and the subsystemAndroid can no longer be started. What should I do?
--------------------------
C:UsersWilheAppDataRoamingMicrosoftInternet ExplorerQuick StartUser
PinnedTaskbar991EX. Ink
The specified path does not exist.

Check the path and try again.
---------------------------

<img width="447" alt="image" src="https://github.com/MustardChef/WSABuilds/assets/38190757/219e17dd-acaf-488d-8abb-752ecd74f28c">

**Comment by MustardChef** [comment]
@5a5ch4 I need more information.
Can you open Android apps from the Start Menu? Do they run?
System Specs?
What app were you trying to open?
